Lib/logging/__init__.py

```diff
@@ -339,9 +339,15 @@ def __init__(self, name, level, pathname, lineno,
         self.stack_info = sinfo
         self.lineno = lineno
         self.funcName = func
-        self.created = ct
-        self.msecs = int((ct - int(ct)) * 1000) + 0.0  # see gh-89047
-        self.relativeCreated = (self.created - _startTime) * 1000
+        self.created = ct / 1e9  # ns to float seconds
+
+        # x // 10**n: Drop the rightmost n digits
+        # (x // 10**n) * 10**n: "Change" the rightmost n digits to 0s.
+        # Eg: 55_123_456_789 --> 123_456_789 --> 123
```

Comment:
This isn't quite an example of the preceding comment lines - the last step of division by 1_000_000 is not in the comment lines above, so this line is confusing.

Reply:
Updated 👍